parser/parser.py

```diff
@@ -0,0 +1,238 @@
+from lark import Lark, Tree, Token
+
+import sys
+sys.path.append("../")
+import python.kgcl
+import python.ontology_model
+from pathlib import Path
+
+def id_generator():
+    id = 0
+    while True:
+        yield id
+        id += 1
+
+path = Path(__file__).parent
+kgcl_parser = Lark.open(str(path) + "/kgcl.lark", start='expression')
+id_gen = id_generator()
+
+#input may be a set of KGCL statements separated by \n
+def parse(input):
+    statements = input.splitlines()
+    parsed = []
+
+    for s in statements:
+        parsed.append(parse_statement(s))
+        #parsed.append((s,parse_statement(s))) 
+    return parsed 
+
+def parse_statement(input): 
+
+    tree = kgcl_parser.parse(input)
+
+    id = "test_id_" + str(next(id_gen))
+
+    command = tree.data
+    #print("Command: " + command)
+
+    if(command == "rename"): 
```

Comment:
minor: can we write this as more idiomatic python. @jamesaoverton do you have a preferred linter - flake8, black, ..?

Reply:
flake8 and black are good for different things, so Knocean usually uses both. @ckindermann please follow https://github.com/knocean/practises/tree/master/python#style and ask me if you have any questions.

Reply:
@jamesaoverton will do!